cypress/fixtures/analysis.sh

```
@@ -0,0 +1,243 @@
+echo "Manifest (file) GENERATED: ${file}"
+
+#
+# Post manifest.
+code=$(curl -kSs -o ${tmp} -w "%{http_code}" \
+  -H "Authorization: Bearer ${TOKEN}" \
+  -F "file=@${file};type=application/x-yaml" \
+  -H 'Accept:application/x-yaml' \
+  https://${host}/applications/${appId}/analyses)
+if [ ! $? -eq 0 ]
+then
+  exit $?
```

Shellcheck (0.10.0)

[warning] This $? refers to a condition, not a command. Assign to a variable to avoid it being overwritten.

(SC2319)